src/common/api/worker/offline/OfflineStorage.ts

```diff
@@ -499,6 +499,14 @@ AND NOT(${firstIdBigger("elementId", range.upper)})`
 		}
 	}
 
+	async deleteWholeList<T extends ListElementEntity>(typeRef: TypeRef<T>, listId: Id): Promise<void> {
+		await this.lockRangesDbAccess(listId)
+		await this.deleteRange(typeRef, listId)
+		const { query, params } = sql`DELETE FROM list_entities WHERE listId = ${listId}`
```

Comment:
you also need to delete the ranges for that list

Reply:
```typescript
await this.deleteRange(typeRef, listId)
```

Is this supposed to do it already?

Reply:
Sorry yes!